contributing.md

```diff
@@ -3,10 +3,9 @@
 Azure Active Directory SDK projects welcomes new contributors.  This document will guide you
 through the process.
 
-### CONTRIBUTOR LICENSE AGREEMENT
+### CONTRIBUTOR CONDUCT
 
-Please visit [https://cla2.msopentech.com/](https://cla2.msopentech.com/) and sign the Contributor License
-Agreement.  You only need to do that once. We can not look at your code until you've submitted this request.
+This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact opencode@microsoft.com with any additional questions or comments.
```

Comment:
@hamiltonha Should we include the CLA Agreement link as well?

https://cla.microsoft.com/